BeagleBone_black/src/motor_control/motor.hpp

```diff
@@ -5,16 +5,14 @@
  * Description:
```

Comment:
Add a short description of what is the purpose of this file. Make sure to mention both the `Motor` class and `MotorsRpm` struct.

Reply:
The 'Motor' class is going to be completely redesigned and replaced with a 'Communicator' class and a 'Controller' class. I am going to remove the motor class before the next prototyping session and replace it with the aforementioned classes, so for now a description is unnecessary.